client/ayon_houdini/api/plugin.py

```diff
@@ -362,6 +362,25 @@ def apply_settings(self, project_settings):
             setattr(self, key, value)
 
 
+class RenderLegacyProductTypeCreator(HoudiniCreator):
```

Comment:
I was wondering whether this would've been possible with some type of MixIn class that didn't require inheriting from `HoudiniCreator` so it's more explicit that it's just a backward compatible 'fix' on the Creators, but couldn't find any sensible way to still be able to access the upstream `apply_settings`...

Not sure if @iLLiCiTiT has any ideas how to do this in a nicer way.

Reply:
Is `legacy_product_type` used only in `product_type` property?

Reply:
Thanks, that's actually what I did at first.. but my IDE complained because in the parent class `product_type` is defined as a property which means it actually can't be assigned to ;) ... the only reason it works on other classes is because we do it at the class definition and set class attribute:
```python
class MyCreator(Creator):
    product_type = "render"
    ...
```
Which essentially takes it back from a property to just a string attribute... but just from the inheritance from `Creator` it actually isn't.

Reply:
> Thanks, that's actually what I did at first.. but my IDE complained because in the parent class product_type is defined as a property which means it actually can't be assigned to ;)

Yeah, but it is abstract property, meaning it has to be overriden. I would not worry about IDE if you're handling backwards compatibility to avoid exactly what you did...

Reply:
Tweaked it with [`7599e07` (#214)](https://github.com/ynput/ayon-houdini/pull/214/commits/7599e07e4b163c1a6d9840efab852bd42ac9f2c5)